runtime/cere-dev/src/lib.rs

```diff
@@ -1254,6 +1254,7 @@ impl pallet_ddc_verification::Config for Runtime {
 	type PalletId = VerificationPalletId;
 	type WeightInfo = pallet_ddc_verification::weights::SubstrateWeight<Runtime>;
 	type ClusterManager = pallet_ddc_clusters::Pallet<Runtime>;
+	type ClusterValidator = pallet_ddc_clusters::Pallet<Runtime>;
 	type NodeVisitor = pallet_ddc_nodes::Pallet<Runtime>;
 	type PayoutVisitor = pallet_ddc_payouts::Pallet<Runtime>;
 	type AuthorityId = ddc_primitives::sr25519::AuthorityId;
```

Comment:
You should include your `pallet_ddc_clusters::migration::MigrateToV3<Runtime>` to the `Migrations` tuple of the runtime,  i.e. there should be the entry of ```type Migrations = (pallet_ddc_clusters::migration::MigrateToV3<Runtime>)``` in the `runtime/cere-dev/src/lib.rs` file